### ISSUE TITLE
[ts][pixi-v8] Spine objects isn't destroyed from memory

### ISSUE THREAD

**Issue by nikollaidemchenko** [EsotericSoftware/spine-runtimes#2678]
this.gpuSpineData from SpinePipe keeps links to spines after they was destroyed

This method wasn't called.

```
	destroyRenderable (spine: Spine) {
		// TODO remove the renderable from the batcher
		this.gpuSpineData[spine.uid] = null as any;
	}
```

Fix: 

add 

```
    if (!Object.values(gpuSpine.slotBatches).length) {
      spine.on('destroyed', this.destroyRenderable.bind(this, spine));
    }
```
    to addRenderable method
    
```
  addRenderable(spine: Spine, instructionSet: InstructionSet) {
    const gpuSpine = (this.gpuSpineData[spine.uid] ||= { slotBatches: {} });

    if (!Object.values(gpuSpine.slotBatches).length) {
      spine.on('destroyed', this.destroyRenderable.bind(this, spine));
    }
    ...
    }
```

add spine.off('destroyed') to destroyRenderable method

```
  destroyRenderable(spine: Spine) {
    this.gpuSpineData[spine.uid] = null as any;
    spine.off('destroyed');
  }
```

**Comment by davidetan** [comment]
I was already fixing the memory leak this morning, see [this commit](https://github.com/EsotericSoftware/spine-runtimes/commit/d0a93b3713ecb55c9285e6a82706bb1a8fe7bed0).

I wanted to test it a little more before merging. I'll probably release it today 👍

**Comment by davidetan** [comment]
We released this in 4.2.65. Even though we were already addressing the issue, I appreciate you reporting this.
